tests/test_cli_credentials.py

```
@@ -0,0 +1,242 @@
+    def test_set_key_interactive(self, runner):
+        """set-key with interactive prompt stores the key."""
+        result = runner.invoke(
+            cli, ["set-key", "FAL_KEY"],
+            input="fal_secret_value\n",
+        )
+        assert result.exit_code == 0
+        assert "FAL_KEY" in result.output
+        assert "saved" in result.output
+        assert load_key("FAL_KEY") == "fal_secret_value"
+
+    def test_set_key_stdin(self, runner, tmp_path, monkeypatch):
```

Ruff (0.15.0)

[warning] Unused method argument: `tmp_path`

(ARG002)